src/affiliations/admin.py

```diff
@@ -1,8 +1,9 @@
 """Admin config for the affiliations service."""
 
 # Third-party dependencies:
-from django.contrib import admin
 from django import forms
+from django.contrib import admin
+from unfold.admin import ModelAdmin # type: ignore
```

Comment:
Did you check to see if there are type hints for django-unfold?

Reply:
Discussed on slack. I have not found type hinting for unfold; however, I could be missing it. I did some additional digging and did not find anything that resolved the original error. Since it might not be supported, the above is the only fix I found.